components/ftx/browser/ftx_pref_utils.cc

```diff
@@ -5,15 +5,23 @@
 
 #include "brave/components/ftx/browser/ftx_pref_utils.h"
 
+#include <string>
+
 #include "brave/components/ftx/common/pref_names.h"
+#include "brave/components/l10n/browser/locale_helper.h"
```

Comment:
maybe we need to add deps for this including?

Reply:
thanks, fixed